android/app/src/main/res/layout/upcoming_dialogbox.xml

```diff
@@ -9,42 +10,42 @@
         android:layout_height="@dimen/upcoming_title_height"
         android:textSize="@dimen/heading_text_size"
         android:layout_marginLeft="@dimen/upcoming_margin_left"
-        android:layout_marginTop="@dimen/layout_margin_large"
         android:textStyle="bold"
         android:textColor="@color/black"
         android:id="@+id/upcoming_Session_title"/>
-    <RelativeLayout
+
+    <LinearLayout
+        android:id="@+id/upcoming_session_details"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginTop="@dimen/layout_margin_medium">
 
         <ImageView
             android:id="@+id/track_image_drawable"
             android:layout_width="@dimen/item_image_view_small"
             android:layout_height="@dimen/item_image_view_small"
             android:layout_marginLeft="@dimen/layout_margin_medium"
-            android:layout_marginTop="@dimen/layout_margin_medium"
             android:contentDescription="@string/item_track" />
 
         <TextView
             android:id="@+id/upcoming_session_textview"
             android:layout_width="match_parent"
-            android:textSize="@dimen/text_size_extra_large"
-            android:layout_toRightOf="@id/track_image_drawable"
+            android:layout_height="wrap_content"
             android:layout_marginLeft="@dimen/upcoming_margin_left"
             android:layout_marginRight="@dimen/layout_margin_large"
-            android:layout_marginTop="@dimen/layout_margin_medium"
+            android:layout_toRightOf="@id/track_image_drawable"
```

Comment:
You changed RelativeLayout to LinearLayout. `layout_toRightOf` has no meaning in LinearLayout